docs/sphinx/conf.py

```diff
@@ -23,7 +23,7 @@
 
 # -- Project information -----------------------------------------------------
 
-project = 'NVIDIA CUDA Quantum'
+project = 'NVIDIA CUDA-Q'
 copyright = '2023, NVIDIA Corporation & Affiliates'
```

Comment:
Just happened to notice that this since your change was right above it ... this still shows 2023 copyright.